package.json

```diff
@@ -15,7 +15,9 @@
     "next": "13.4.8",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^6.0.2"
+    "remixicon": "^4.5.0",
```

Comment:
What is remixicon and where do you use it?

Reply:
we use it for our icons